package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oak-js-library",
-  "version": "0.2.0",
+  "version": "0.3.0",
```

Comment:
bumping version cause we migrated up to the new double storage maps, so the old observer functions needed to be changed. that was previously added, but i forgot to bump version. along with the rpc changes, i think new version bump prob needed?